dev/packages/alpha/mysql-0.0.2/dataset/slowlog/manifest.yml

```diff
@@ -0,0 +1,24 @@
+id: slowlog
+title: MySQL slowlog logs
+release: ga
+type: logs
+streams:
+- input: logs
+  vars:
+  - name: paths
+    type: text
+    title: Slowlog paths
+    multi: true
+    required: true
+    show_user: true
+    default:
+    - /var/log/mysql/mysql-slow.log*
+    - /var/lib/mysql/{{.builtin.hostname}}-slow.log
+    os.darwin:
```

Comment:
The platform flags will not work yet but please keep them in as we plan to support them later.

Reply:
I adjusted all config properties with path variables.